src/protocols/ColorManagement.cpp

```diff
@@ -0,0 +1,571 @@
+#include "ColorManagement.hpp"
```

Comment:
```suggestion
#include "ColorManagement.hpp"
#include "Compositor.hpp"
```

Reply:
why? seems to work as is. probably included somewhere else